tmp/build/patch-charts.sh

```diff
@@ -241,14 +241,33 @@ function patchKialiOpenShift() {
     sed -i -e '/server:/ i\
 \    istio_namespace: {{ .Release.Namespace }}' ${HELM_DIR}/istio/charts/kiali/templates/configmap.yaml
   fi
-  sed -i -e '/jaeger:/ a\
-\        namespace: {{ .Release.Namespace }}' ${HELM_DIR}/istio/charts/kiali/templates/configmap.yaml
-  sed -i -e '/grafana:/ a\
-\        service_namespace: {{ .Release.Namespace }}' ${HELM_DIR}/istio/charts/kiali/templates/configmap.yaml
+  sed -i -e '/PROMETHEUS_SERVICE_URL/,+1 d' ${HELM_DIR}/istio/charts/kiali/templates/deployment.yaml
```

Comment:
FYI: This isn't in the latest Kiali deployment. We don't use the env vars anymore by default - everything is in the Kiali CR for the operator. Since the [kiali operator PR](https://github.com/Maistra/istio-operator/pull/87) hasn't been merged yet, this is good, no change needed. We just need to remember to fix this over in that PR (this will cause a conflict) - the kiali templates for the most part will all be deleted and we'll just have a kiali-cr.yaml. The same is true for the code below that changes the configmap with the jaeger and grafana stuff - all of that is going to be ripped out.